src/main/java/frc/robot/poseestimator/helpers/objectListOrganizer.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.poseestimator.helpers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class objectListOrganizer<T> {
+
+	private final Supplier<List<T>> observationSupplier;
+	private final int maxDataIntakeRepeats;
+	private List<T> stackedObservations;
```

Comment:
```suggestion
	private List<T> ObservationStack;
```

Reply:
there is no stack, all ready resolved it.